docs/src/pages/components/tables/CustomPaginationActionsTable.tsx

```diff
@@ -140,9 +140,6 @@ export default function CustomPaginationActionsTable() {
   const [page, setPage] = React.useState(0);
   const [rowsPerPage, setRowsPerPage] = React.useState(5);
 
-  const emptyRows =
```

Comment:
This is important to avoid layout jump when changing the page

Reply:
Are you talking about all that `emptyRows` logic?

Reply:
@oliviertassinari How would one reproduce the layout jump on https://5f98bab8ec5ca100074fe34a--material-ui.netlify.app/components/tables/#custom-pagination-options (original PR)?

Reply:
![jump](https://user-images.githubusercontent.com/3165635/97429158-52430c00-1917-11eb-8e4e-159d2bf23561.gif)